src/main/java/com/redhat/exhort/providers/GradleProvider.java

```diff
@@ -62,6 +62,11 @@ public GradleProvider(Path manifest) {
     super(Type.GRADLE, manifest);
   }
 
+  public enum AnalysisType {
```

Comment:
This is not specific to Gradle but application wide. If you decide to create such enum you should consider using it each time there is a decision between stack and component or at least define it in a common model package.

Reply:
Updated. Thanks